content/issues/2024-12-23-christmas-2024.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Christmas is here 🎄"
+date: "2024-12-24T19:00:00+01:00"
```

Comment:
Will somebody really be working until 7pm on 24th? :sweat_smile:

Reply:
there's a different date in the filename though :eyes:

Reply:
initially it was 23T19 but then I thought 24 was better and I forgot to change the hour... thanks for spotting it!